src/codec/encoder/MayaEncoder.cpp

```diff
@@ -618,6 +621,9 @@ void MayaEncoder::convertGeometry(const prtx::InitialShape& initialShape,
 	}
 	faceRanges.push_back(faceCount); // close last range
 
+	if (faceRanges.size() <= 1)
```

Comment:
added SerializedGeometry::isEmpty() check, return when serializedGeometry is empty instead of using this check now
IsEmpty simply checks if there are faces/vertices and indices available. It does however not check its validity. 
```
bool isEmpty() const {
  if (coords.empty() || counts.empty() || vertexIndices.empty()) {
    return true;
  }
  return false;
}
```